zwave_js_server/model/log_config.py

```diff
@@ -1,4 +1,5 @@
 """Provide a model for the log config."""
+from __future__ import annotations
```

Comment:
We can update `from_dict` to return type `LogConfig` with this.

Reply:
Oh I meant to not include that and stick with the quoted type since that's used elsewhere. Which would we prefer? I'd like to make it consistent across the board and I don't have a particular preference for one way or another

Reply:
We can keep it quoted for now and remove this. We can update everything later. 👍